lib/Sema/TypeCheckAttr.cpp

```diff
@@ -1617,7 +1617,7 @@ void AttributeChecker::checkApplicationMainAttribute(DeclAttribute *attr,
     namelookup::lookupInModule(KitModule, Id_ApplicationDelegate,
                                decls, NLKind::QualifiedLookup,
                                namelookup::ResolutionKind::TypesOnly,
-                               KitModule);
+                               nullptr);
```

Comment:
I thought you defaulted this argument, but if not can you give it an argument label comment?

Reply:
Actually we don't default it, and it can't be null. We assert if it is. I kept this behavior from the old implementation.